.github/workflows/cross-compile-daily-mips.yml

```diff
@@ -0,0 +1,22 @@
+name: Daily Cross Compile Tests
```

Comment:
This change can be dropped since we merge mips into master

Reply:
We would need a fix up commit on `master` to include `mips64el-cross` in `.github/workflows/cross-compile-daily.yml`. Probably not really important as we know it works on `criu-dev` and the daily build will happen on `criu-dev`.